File: Array/21_30/27Solution.py
def targetIndices(nums,target):
        equal=great=less=0
        lst=[]
        for i in range(len(nums)):
            if nums[i]<target:
                less=less+1
            elif nums[i]==target:
                equal=equal+1
            else:
                great=great+1
        for j in range(equal):
            lst.append(less+j)
        return lst
nums=[ int(x.strip()) for x in input().split(",")]
target=int(input())

File: Array/21_30/27Solution.java
class Solution {
    public List<Integer> targetIndices(int[] nums, int target) {
        List<Integer> list=new ArrayList<>();
        int count=0,index=0;
        for(int num:nums){
            if(num==target)
                count++;
            if(num<target)
                index++;
        }
        
        while(count-->0)
            list.add(index++);
        
        return list;
    }
}
